vck/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/IssuerAgent.kt

```diff
@@ -189,6 +189,21 @@ class IssuerAgent(
         return Issuer.IssuedCredential.VcSdJwt(vcInSdJwt, credential.scheme)
     }
 
+    data class DisclosuresAndDigests(
+        val disclosures: Collection<String>,
+        val digests: Collection<String>,
+    )
+
+    private fun CredentialToBeIssued.VcSd.toDisclosuresAndDigests(): DisclosuresAndDigests {
+        val disclosures = claims
+            .map { SelectiveDisclosureItem(Random.nextBytes(32), it.name, it.value) }
```

Comment:
just curious: how `SecureRandom` must this random be? because there is a KMP SecureRandom courtesy of Matthew Nelson. This probably applies to more randoms in VC-K